### DISCUSSION TITLE
installation warnings

### DISCUSSION THREAD

**Discussion by MarekBarski** [NVlabs/instant-ngp#1151] [Q&A] [answered]
Hello
i was try to install instant_ngp but after cmake things i dont have testbed file.
after command:
cmake --build build --config RelWithDebInfo -j 16

i got few yellow wornings:
G:\NGP\instant-ngp\src\triangle_bvh.cu(49): warning #177-D: variable "ngp::MAX_DIST_SQ" was declared but never referenc
ed [G:\NGP\instant-ngp\build\ngp.vcxproj]

CUDACOMPILE : warning #20014-D: calling a __host__ function from a __host__ __device__ function is not allowed [G:\NGP\
instant-ngp\build\ngp.vcxproj]

G:\NGP\instant-ngp\dependencies\tinyexr/tinyexr.h(7381): warning C5208: unnamed class used in typedef name cannot decla
re members other than non-static data members, member enumerations, or member classes [G:\NGP\instant-ngp\build\ngp.vcx
proj]

how i can fix this?

**Comment by Tom94** [comment, marked as answer]
Since a while ago, it's called `instant-ngp.exe`, not testbed. Do you have that one?

**Comment by MarekBarski** [comment]
I was download newest cmake, after rebuild appered instant-ngp.exe and every thing works fine. 
As I understand it, there is no way to burn the maps on the UV model? Just vertex color?
